doc/Command_Index_internal.tex

```diff
@@ -748,7 +740,7 @@ \section{Gregorio\TeX{} Controls}
 \end{argtable}
 
 \macroname{\textbackslash gre@char@cavum}{\#1\#2\#3\#4\#5\#6\#7\#8}{gregoriotex-signs.tex}
-Macro to typeset a "cavum" character.
+Macro to typeset a “cavum” character.
```

Comment:
There are unicode quotes in some places and TeX double backquotes/apostrophes in others.  Should we be consistent?  In the text editor I use, it's not convenient to type unicode quotes, but I will follow whatever convention you want here.